source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1655,9 +1661,6 @@ def _get_groupName(self):
 		else:
 			return super(IAccessible,self)._get_groupName()
 
-	def event_selection(self):

```

Comment:
does removing this break add-on compatibility?

Reply:
No, as event_selection is still available and still does the same thing, I.e. calls event_stateChange, it is just that it has been moved to a base class.